### ISSUE TITLE
in some math operations, like torch.std, keepdim is not 3rd parameter?

### ISSUE THREAD

**Issue by Qoo95** [pytorch/pytorch#31619]
### Some Question
when I try some math operations, like torch.std, keepdim is not its 3rd parameter, but torch.mean is. That make me confusion.
```python
a = torch.randn(3, 3, 3)
torch.mean(a, (1, 2), True)
# tensor([[[-0.2389]], 
#           [[-0.3910]],
#           [[ 0.2703]]])
torch.std(a, (1, 2), True)
# tensor([1.4173, 0.9174, 1.1403])
```

### Another type wrong
in doc, the second item of torch.std_mean, that is torch.std(input, dim, keepdim=False, unbiased=True)

**Comment by ngimel** [comment]
keepdim is a keyword argument, `a.std((1,2), keepdim=True)` produces expected result. Documentation lists `keepdim` as a keyword argument, so as far as current documentation is concerned behavior is consistent. It's confusing through that third positional argument is ignored for torch.std - would you like to submit a PR fixing that?

**Comment by Qoo95** [comment]
I just check the declaration of torch.std and torch.std_mean, the third positional argument is unbiased. I do not know which is better, modifying doc or changing the order of parameters?

**Comment by ngimel** [comment]
Changing the order of parameters will break backward compatibility, and this is definitely not a good enough reason to do it, so changing the docs is the preferred way.
